aicsimageio/readers/reader.py

```diff
@@ -56,38 +101,16 @@ def guess_dim_order(shape: Tuple[int]) -> str:
         dim_order: str
             The guessed dimension order.
         """
-        pass
-
-    @classmethod
-    def assert_reader_supports_image(cls, image: types.ImageLike) -> bool:
-        """
-        Asserts that the provided image like object is supported by the current Reader.
-
-        Parameters
-        ----------
-        image: types.ImageLike
-            The filepath or array to validate as a supported type.
-
-        Returns
-        -------
-        supported: bool
-            Boolean indicated if the provided data is or is not supported by the
-            current Reader.
-
-        Raises
-        ------
-        TypeError: Invalid type provided to image parameter.
-        """
-        pass
+        return DEFAULT_DIMENSION_ORDER[len(DEFAULT_DIMENSION_ORDER) - len(shape) :]
```

Comment:
could shape be longer than default dimension order?

Reply:
Good question! Yes it _could_ happen, but most readers will have their own image dimension guesser / usage of metadata to determine what the dimensions are. So I am not personally concerned about it?

Reply:
just that if it happens users will get some unexpected thing returned here

Reply:
Hmmmm what is the recommendation here? I could add dimensions of `"?"` to the end?